corehq/apps/locations/models.py

```diff
@@ -468,6 +468,39 @@ def get_descendants(self, include_self=False, **kwargs):
             where, **kwargs
         )
 
+    @staticmethod
+    def get_bulk_ancestors(domain, location_ids, **kwargs):
```

Comment:
What does `kwargs` do here nothing is passed in there but it is passed onto the next function? Its just an empty dict?

Reply:
yea its just empty dict. anyway. I am shifting this function to report there i think it wont make sense to pass it. I added in as I made this function a part of SQLLocation class to allow others to use it.